src/utils/daos_hdlr.c

```diff
@@ -2751,18 +2761,16 @@ dm_connect(bool is_posix_copy,
 		ca->cont_layout = dpe_vals[0];
 		ca->cont_oid = dpe_vals[1];
 
-		props = daos_prop_alloc(2);
-		if (props == NULL) {
-			fprintf(stderr, "Failed to allocate prop (%d)", rc);
-			D_GOTO(out, rc);
-		}
-		props->dpp_entries[0].dpe_type = ca->cont_prop_layout;
-		props->dpp_entries[0].dpe_val = ca->cont_layout;
-
-		/* only set max oid on created dest containers
-		 * if this is a non-posix copy
-		 */
 		if (!is_posix_copy) {
+			props = daos_prop_alloc(2);
+			if (props == NULL) {
+				fprintf(stderr, "Failed to allocate prop (%d)",
+					rc);
+				D_GOTO(out, rc);
```

Comment:
@dsikich Just noticed... daos_prop_alloc doesn't return an rc
```suggestion
			props = daos_prop_alloc(2);
			if (props == NULL) {
				fprintf(stderr, "Failed to allocate prop\n");
				D_GOTO(out, rc = -DER_NOMEM);
```